py/tests/test_helpers.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python3
+
+from pykms import *
+from argparse import *
+
+class ArgumentParserHelper(ArgumentParser):
```

Comment:
Yes, it is overly simplified, but I don't think it would be hard extend it to support arbitrary number of connecector, crtcs, and planes. Anyway, maybe it is better to drop it now.

What about the three first patches? Would you merge them if I would send them separately.

Reply:
Yes, I can merge the first three. No need to send, I can just pick those.